pkg/virt-controller/watch/vmi.go

```diff
@@ -2139,6 +2139,9 @@ func (c *VMIController) getFilesystemOverhead(pvc *k8sv1.PersistentVolumeClaim)
 	if !ok {
 		return "0", fmt.Errorf("Failed to convert CDIConfig object %v to type CDIConfig", cdiConfigInterface)
 	}
+	if cdiConfig.Status.FilesystemOverhead == nil {
+		return "0", fmt.Errorf("Failed to get FilesystemOverhead from CDIConfig's status")
+	}
```

Comment:
Would like to understand more about why/how system got in this state in order to better understand if we should handle the result of `getFilesystemOverhead` differently here:  https://github.com/kubevirt/kubevirt/blob/ff5cef9bd4f062a0788d3d22d9bc528073e3c704/pkg/virt-controller/watch/vmi.go#L2090-L2098